pkg/gcptarget/matcher.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcptarget
+
+import (
+	"fmt"
+
+	"github.com/gobwas/glob"
+)
+
+var ErrInvalidReview = fmt.Errorf("unexpected type of review, expect map[string]interface{}")
+var ErrInvalidAncestryPath = fmt.Errorf("unexpected type of ancestry path in review object")
+
+type matcher struct {
+	include []string
```

Comment:
I'd prefer to have the matcher struct use `ancestries` and `excludedAncestries` to match the preferred language and in case we want to add additional fields later. What would you think about that?

Reply:
Done